src/dividend/XdShare.sol

```diff
@@ -11,43 +11,43 @@ import {ReentrancyGuardUpgradeable} from
 import {SafeERC20, IERC20} from "openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol";
 
 /**
- * @title xdShare Contract
- * @dev This contract acts as a wrapper over the dShare token, providing additional functionalities.
- *      It serves as a reinvestment token that uses dShare as the underlying token.
+ * @title XdShare Contract
+ * @dev This contract acts as a wrapper over the DShare token, providing additional functionalities.
+ *      It serves as a reinvestment token that uses DShare as the underlying token.
  *      Additionally, it employs the ERC4626 standard for its operations.
- *      If TokenManager is not used, make sure that dShare will never split.
- * @author Dinari (https://github.com/dinaricrypto/sbt-contracts/blob/main/src/xdShare.sol)
+ *      If TokenManager is not used, make sure that DShare will never split.
+ * @author Dinari (https://github.com/dinaricrypto/sbt-contracts/blob/main/src/XdShare.sol)
  */
 // slither-disable-next-line missing-inheritance
-contract xdShare is Initializable, ERC4626, OwnableUpgradeable, ReentrancyGuardUpgradeable {
+contract XdShare is Initializable, ERC4626, OwnableUpgradeable, ReentrancyGuardUpgradeable {
```

Comment:
`XDShare`

Reply:
detect `naming-convention` it will flag for variable like _perOrderFee, starting with _